model/semconv/v1.7.0/nonstandard.go

```diff
@@ -0,0 +1,22 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package semconv // import "go.opentelemetry.io/collector/model/semconv/v1.7.0"
+
+const (
+	OtelLibraryName       = "otel.library.name"
+	OtelLibraryVersion    = "otel.library.version"
+	OtelStatusCode        = "otel.status_code"
+	OtelStatusDescription = "otel.status_description"
+)
```

Comment:
I assume this is an exact copy from 1.6.1 directory, right?

Reply:
Yes, it is. I didn't find any changes on these definitions up to 1.10.0

Reply:
If we managed to not have these (especially because they are manually written), is this a sign that we don't need them?

Reply:
If we managed to not have these (especially because they are manually written), is this a sign that we don't need them?

Reply:
If we managed to not have these (especially because they are manually written), is this a sign that we don't need them?

Reply:
I can add a test verifying that the files are created

Reply:
added the test https://github.com/open-telemetry/opentelemetry-collector/pull/5106